extensions/amp-access/0.1/amp-login-done.html

```diff
@@ -11,8 +11,53 @@
       margin: 0;
     }
 
+    header {
+      padding: 16px;
+      display: -webkit-box;
+      display: -webkit-flex;
+      display: -ms-flexbox;
+      display: flex;
+      -webkit-box-orient: horizontal;
+      -webkit-box-direction: normal;
+      -webkit-flex-direction: row;
+      -ms-flex-direction: row;
+      flex-direction: row;
+      -webkit-flex-wrap: nowrap;
+      -ms-flex-wrap: nowrap;
+      flex-wrap: nowrap;
+      -webkit-box-align: center;
+      -webkit-align-items: center;
+      -ms-flex-align: center;
+      align-items: center;
+    }
+
+    .logo,
+    .logo svg {
+      width: 40px;
+      height: 40px;
+    }
+
+    .logo path {
+      fill: #bdbdbd;
+    }
+
+    .logo-text {
+      color: #bdbdbd;
+      font-family: Roboto, verdana, arial, sans-serif;
+      font-size: 14px;
+      letter-spacing: 2px;
+      padding-left: 0;
+      padding-top: 18px;
+      text-transform: uppercase;
+      padding: 10px 15px;
```

Comment:
This overrides the previous padding setting on the same selector - drop the ones above?